source/instructors/setupcourses/batch-assignment-update.rst

```diff
@@ -0,0 +1,50 @@
+.. meta::
+   :description: Batch Assignment Settings Update
+
+
+.. _batch-assignment-update:
+
+Batch Assignment Update
+=======================
+
+You can do the batch update of some assignments settings like Start date/time of the assignment, Closing date/time of the assignment, Action when assignment closes and Prime Assignment Containers settings using a CSV file.
+
+To batch update the assignments settings, follow these steps:
+
+1. Go to **Bulk Settings** area and press **Open Updater** button from Update Assignment Setting
+
+   .. image:: /img/bulk-setting-area.png
+      :alt: Bulk Setting Area
+
+
+2. Select the CSV file in which you have defined all the required settings and press **Detect CSV format** button
+
+   .. image:: /img/select-csv-batch-update.png
+      :alt: Select csv Batch Update
+
+
+   You can use the below CSV as a starting point and customize it as per your needs
```

Comment:
looks OK but the wording: You can use the below CSV as a starting point and customize it as per your needs      seems wrong as indicates that they can download it .  Would suggest the words maybe better to say :  Below is example of a csv file.